src/core/ext/filters/client_channel/lb_policy/xds/xds_override_host.cc

```diff
@@ -181,8 +233,23 @@ LoadBalancingPolicy::PickResult XdsOverrideHostLb::Picker::Pick(
     return PickResult::Fail(absl::InternalError(
```

Comment:
Still need to move this check (lines 232-235) down to line 245, so that we're not checking that the child picker is non-null unless we're actually going to use it.